libsolc/libsolc.cpp

```diff
@@ -229,12 +229,12 @@ string compile(StringMap const& _sources, bool _optimize, CStyleReadFileCallback
 
 string compileMulti(string const& _input, bool _optimize, CStyleReadFileCallback _readCallback = nullptr)
 {
-	Json::Reader reader;
+	std::string errs;
```

Comment:
The `std` namespace is imported in this file.

Reply:
Also it could be just `errors` to avoid abbreviations.

Reply:
Sure! I really love explicitly written namespaces - but I will change my coding style for you. But `errs` should really be renamed to `errors`.